test/expected/pg_dump.out

```diff
@@ -591,24 +619,52 @@ SELECT get_sqlstate('SELECT timescaledb_post_restore()');
 (1 row)
 
 drop function get_sqlstate(TEXT);
-\c postgres :ROLE_SUPERUSER
---need to shutdown workers to use db as template
--- there should be no sessions for that database
--- columns are explicit so the output is the same on PG12 and PG13 since PG13 has extra column leader_pid
-SELECT datid,datname,pid,usesysid,usename,application_name,client_addr,client_hostname,client_port,backend_start,xact_start,query_start,state_change,wait_event_type,wait_event,state,backend_xid,backend_xmin,query,backend_type FROM pg_stat_activity WHERE datname = :'TEST_DBNAME';
- datid | datname | pid | usesysid | usename | application_name | client_addr | client_hostname | client_port | backend_start | xact_start | query_start | state_change | wait_event_type | wait_event | state | backend_xid | backend_xmin | query | backend_type 
--------+---------+-----+----------+---------+------------------+-------------+-----------------+-------------+---------------+------------+-------------+--------------+-----------------+------------+-------+-------------+--------------+-------+--------------
-(0 rows)
-
-CREATE DATABASE db_dump_error WITH TEMPLATE :TEST_DBNAME;
---now test functions for permission errors
-\c  db_dump_error :ROLE_DEFAULT_PERM_USER_2
-\set ON_ERROR_STOP 0
+-- Check that the extension can be copied from an existing database
+-- without explicitly installing it. Stop background workers since we
+-- cannot have any backends connected to the database when cloning it.
+\c :TEST_DBNAME :ROLE_SUPERUSER
 SELECT timescaledb_pre_restore();
-ERROR:  must be owner of database db_dump_error
-SELECT timescaledb_post_restore();
-ERROR:  must be owner of database db_dump_error
-\set ON_ERROR_STOP 1
---drop db
+ timescaledb_pre_restore 
+-------------------------
+ t
+(1 row)
+
+SELECT wait_worker_counts(0,0);
```

Comment:
Why do we need to wait on workers to exit now that template0 is used instead? I thought the intention was to use a database that doesn't have background workers so that we don't need to stop and wait for background workers.